src/components/cutSnippetToFiveLines.ts

```diff
@@ -0,0 +1,12 @@
+export function cutSnippetToFiveLines(code: string): string {
+    const codeArray = code.split("\n");
+    if (codeArray.length < 6) {
+        return code;
+    } else {
```

Comment:
good that you extracted this function so it can be reused for each code snippet submission